.github/workflows/test_and_deploy.yml

```diff
@@ -34,6 +34,9 @@ jobs:
           python-version: "3.9"
 
     steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
```

Comment:
The `actions/checkout` docs are vague. I was wondering if one could specify the files manually instead of just everything tracked by Git LFS - in case we don't need all of them for the CI.

See this issue regarding my concerns on bandwith https://github.com/actions/checkout/issues/834

Reply:
Yeah I'd rather we continued to host all large files at https://gin.g-node.org/BrainGlobe.